frontend/src/views/Page404.vue

```diff
@@ -0,0 +1,25 @@
+<template>
```

Comment:
Do we still need this?

Reply:
Just in case user type in some wrong URL. I incline to keep the 404 page.